inc/featured-media.php

```diff
@@ -157,109 +132,51 @@ function largo_get_featured_image_hero($post = null, $classes = '') {
 		'the_post' => $the_post
 	);
 
-	ob_start();
-	largo_render_template('partials/hero', 'featured-image', $context);
-	$ret = ob_get_clean();
-
-	return $ret;
-}
-
-
-/**
- * Prints DOM for an embed code hero.
- *
- * @since 0.5.1
- * @see largo_get_featured_embed_hero()
- *
- * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
- * @param String $classes Optional. Class string to apply to outer div.hero
- */
-function largo_featured_embed_hero($post = null, $classes = '') {
-
-	echo largo_get_featured_embed_hero($post,$classes);
-
-}
-
-/**
- * Returns DOM for an embed code hero.
- *
- * @since 0.5.1
- *
- * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
- * @param String $classes Optional. Class string to apply to outer div.hero
- */
-function largo_get_featured_embed_hero($post = null, $classes = '') {
-
-	$the_post = get_post($post);
-	$featured_media = largo_get_featured_media($the_post->ID);
-
-	if( !in_array($featured_media['type'], array('embed-code', 'video')) ) {
-		return;
+	switch( $featured_media['type'] ) {
+		case 'video':
+			$template_slug = 'video';
```

Comment:
`partials/hero-featured-video` is not a file that exists.

This line should use `$template_slug = 'embed'`, to match the old code:

```php
if (in_array($featured_media['type'], array('embed-code', 'video'))) {
	$ret = largo_get_featured_embed_hero($post->ID,$classes);
```